app/controllers/cases_controller.rb

```diff
@@ -340,6 +343,40 @@ def close
     set_permitted_events
   end
 
+  def process_date_responded
+    authorize @case, :can_close_case?
+    @case.prepare_for_respond
+    if !@case.update respond_date_params
+      render :close
+    else
+      @team_collection = CaseTeamCollection.new(@case)
+      @case.update(late_team_id: @case.responding_team.id)
+      render :closure_outcomes
+    end
+  end
+
+  def closure_outcomes
```

Comment:
Is this action being used anywhere? There's no route for it and it doesn't work when visiting it via the browser.